teps/0044-data-locality-and-pod-overhead-in-pipelines.md

```diff
@@ -581,8 +624,8 @@ Pros:
 * Allows Pipeline to run multiple Tasks in one pod without having to support all of a Pipeline's functionality in a pod
 
 Cons:
-* Will need to update our entrypoint logic to allow for containers running in parallel
-  * We could (at least initially) only support sequential groups
+* No reason to introduce a new grouping syntax when we plan to have [Pipelines in Pipelines](./0056-pipelines-in-pipelines.md).
```

Comment:
We don't have it yet in the core api - but hopefully soon enough :)

Reply:
yes, pipelines in pipelines is coming very soon 😁